modules/cnf-configuring-node-groups-for-the-numaresourcesoperator.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+
+[id="cnf-configuring-node-groups-for-the-numaresourcesoperator_{context}"]
+= Configuring NUMA Resource Operator polling
```

Comment:
the subject here is wrong: the numaresources operator in turn configures agents on the node (daemonsets, resource-topology-exporter) which in turn do the polling according to the settings documented here.
Everything else looks fine.